### DISCUSSION TITLE
Elastic 7.12.0 Enhancements and Securityonion

### DISCUSSION THREAD

**Discussion by faisalusuf** [Security-Onion-Solutions/securityonion#4030] [Unsupported Versions] [answered]
Hi,

With the latest release of Elastic Security 7.12.0 many enhancements are available including single agent endpoint security(Antivirus and EDR), unified policing and integration, threat intelligence as beats modules along with support for log ingestion from major security vendors, and centralized fleet a new management for elastic agent. This way NSM, SIEM, and endpoint detection and response all can be achieved.

How this can be leveraged within SO or is it even possible with the release of 2.3.50 ? Is there any risk in experimenting or adding new Kibana app within SO?

Thanks
Faisal.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/issues/3599

None of the endpoint pieces will be supported right now as they require Kibana as well as Elastic to be exposed directly to the network. To secure this we will need to support Elastic Auth which we do not support at this time. Unfortunately enabling auth is not a simple thing to do when it comes to automation and supporting currently installed SO instances.

**Reply by faisalusuf** [reply]
Thanks for advice and very logical to keep the security first approach toward new features. 

Regards,
Faisal.
